README.md

```
@@ -18,14 +18,22 @@ Features
  - Automatic Payload Serialization
  - Basic Auth
- - Client Side Certificate Auth (SSL)
- - Request "Download"
- - Request "Templates"
- - Parallel Request (via curl_multi)
- - PSR-3: Logger Interface
- - PSR-7: HTTP Message Interface
- - PSR-17: HTTP Factory Interface
- - PSR-18: HTTP Client Interface
+ - Bearer Token Auth
+  - Client Side Certificate Auth (SSL)
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 1; Actual: 2

(MD005, list-indent)